### DISCUSSION TITLE
Como comentar linha de script por atalho.

### DISCUSSION THREAD

**Discussion by syspone** [dbeaver/dbeaver#34158] [Q&A] [answered]
Gostaria de saber como faço pra comentar linha de script por atalho.

segue exemplo;

![image](https://github.com/dbeaver/dbeaver/assets/48097255/f79fd2fa-02af-4924-ab00-fb9a10215ee4)

**Comment by E1izabeth** [comment, marked as answer]
You can use `Ctrl+/` (`⌃/`  for Mac) shortcut
Here you can find more shortcuts https://dbeaver.com/docs/dbeaver/Shortcuts/#sql-editor
